packages/rps/package.json

```diff
@@ -15,11 +15,13 @@
     "build": "run-s clearContracts build:typescript build:webpack",
     "clearContracts": "rm -rf ./build/contracts",
     "contracts:compile": "node ./bin/compile.js",
-    "lint:check": "tslint -c tslint.json -p .",
-    "lint:write": "tslint -c tslint.json -p . --fix",
+    "lint:check": "run-s eslint:check prettier:check",
+    "lint:write": "run-s eslint:write prettier:write",
+    "eslint:check": "eslint \"**/*.{js,ts,tsx}\" --cache",
+    "eslint:write": "eslint \"**/*.{js,ts,tsx}\" --fix",
     "postinstall": "rm -f node_modules/web3/index.d.ts",
-    "prettier:check": "prettier --check '**/*.{scss,json,js,ts,tsx}' --ignore-path '.gitignore'",
-    "prettier:write": "prettier --write '**/*.{scss,json,js,ts,tsx}' --ignore-path '.gitignore'",
+    "prettier:check": "prettier --check '**/*.{scss,json}' --ignore-path '.gitignore'",
```

Comment:
I think `eslint` is running `prettier` so we shouldn't need special `prettier:check` scripts

Reply:
ESLint, from what I could manage to figure out, can't run on `scss` and `json` files
I want to run Prettier on those files though :)